recipes/inn-interface/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "INN-interface" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com//NREL/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3322f2aa168109aab764e1f8559ebd8cbf29ea4b57d3032796e13770b4bad6e7
+
+build:
+  skip: true  # [win or py<35]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - git
+    - g2aero
+    - h5py
+    - matplotlib-base
+    - numpy
+    - tensorflow >=2.7.0
+    - wisdem
+
+test:
+  imports:
+    - INN_interface
+
+about:
+  home: https://github.com/NREL/INN-interface
+  summary: 'Invertible neural network (INN) interface for WISDEM'
+  description: |
+    This software contains the invertible neural network (INN) 
+    model for airfoil design and the high-level API to interface 
+    with the Wind-Plant Integrated System Design and Engineering 
+    odel (WISDEM). Individually, the model can generate new 
+    airfoil shapes with desired performance characteristics. 
+    Coupled with WISDEM, the software enables full 3D blade 
+    design that consider aerodynamics, structural requirements, 
+    and cost.
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - aglaws
```

Comment:
@jefalon since you submitted this recipe, did you want to be listed as a maintainer too?

Reply:
I'm just helping Andrew get this package onto conda-forge since I have experience doing it in the past, so I don't need to be a maintainer.